packages/actions/index.ts

```diff
@@ -584,6 +584,12 @@ export const rejectPendingSettlement = (pendingSettlement: PendingUnilateral) =>
   }
 }
 
+export const setInitialHomeLoad = () => {
```

Comment:
This would normally be named something like `clearInitialHomeLoad` or `resetInitialHomeLoad` since you are setting that boolean to false....

Should we consider `setInitialHomeLoad(startScreen)` where instead of a boolean we could specify which screen to start on? For now that would only support 'Friends'. Not sure if that would be useful or not.